client/src/components/Header/components/auth-or-profile.tsx

```diff
@@ -21,14 +21,16 @@ const AuthOrProfile = ({ user }: AuthOrProfileProps): JSX.Element => {
     );
   } else {
     return (
-      <Link className='avatar-nav-link' to={`/${user.username as string}`}>
-        <AvatarRenderer
-          isDonating={isUserDonating}
-          isTopContributor={isTopContributor}
-          picture={user.picture}
-          userName={user.username}
-        />
-      </Link>
+      <div>
+        <Link className='avatar-nav-link' to={`/${user.username as string}`}>
+          <AvatarRenderer
+            isDonating={isUserDonating}
+            isTopContributor={isTopContributor}
+            picture={user.picture}
+            userName={user.username}
+          />
+        </Link>
+      </div>
```

Comment:
Because you have updated the structure of the HTML here, you'll need to update the [related test](https://github.com/freeCodeCamp/freeCodeCamp/blob/2154b40ab623810ca609d6b7806a3fc20bf1528f/client/src/components/Header/header.test.tsx#L82) to look for the correct element in the DOM structure.

Reply:
Hello @Sboonny 
- Done all the changes as said